ProjFS.Mac/PrjFSLib/PrjFSLib.h

```diff
@@ -89,6 +89,7 @@ typedef enum
 {
     PrjFS_UpdateType_Invalid                        = 0x00000000,
     
+    PrjFS_UpdateType_AllowDirtyData                 = 0x00000002,
```

Comment:
Why do we need this flag?

Reply:
Good point, we do not.

I added it here because it was already present in `PrjFSLib.Mac.UpdateType`, but I think rather than adding it here I can remove it from `UpdateType`